server/app/lib/actions/fusor/deployment/rhev/ose_launch.rb

```diff
@@ -57,10 +57,10 @@ def run
                                                :proxy => Domain.find(1).proxy
                                              })
             if subdomain.valid?
+              ::Fusor.log.debug "====== OSE wildcard subdomain is not valid, it might conflict with a previous entry. Skipping. ======"
```

Comment:
Is this message correct? So ```subdomain.valid?``` returns true if it exists. So wouldn't it say:

```
OSE wildcard subdomain might conflict with a previous entry. Skipping.
```

It is valid, just a duplicate. Maybe I'm being pedantic.